string header
uint16 version
uint32 packet_number
time stamp